fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/FDBDatabase.java

```diff
@@ -766,14 +766,23 @@ private Transaction createTransaction(@Nonnull FDBRecordContextConfig config,
         return transaction;
     }
 
+    /**
+     * Create an {@link FDBDatabaseRunner} for use against this database.
+     * @param contextConfigBuilder options for contexts opened by the new runner
```

Comment:
It might be worth clarifying that the builder passed in will be the same object used in the runner, so future changes to the state of the argument will also affect the values as seen in the runner (as it's running).

Reply:
Added